daemon/logger/ring.go

```diff
@@ -93,7 +93,9 @@ func (r *RingLogger) Close() error {
 		}
 
 		if err := r.l.Log(msg); err != nil {
-			logrus.WithField("driver", r.l.Name()).WithField("container", r.logInfo.ContainerID).Errorf("Error writing log message: %v", r.l)
```

Comment:
Nice catch. Can we put the error in a "WithError" instead?

Reply:
That would be better. I updated to use WithError.